arrabiata/src/main.rs

```diff
@@ -92,6 +92,42 @@ pub fn main() {
         // FIXME: Check twice the updated commitments
         env.compute_and_update_previous_commitments();
 
+        // FIXME:
+        // Absorb all commitments in the sponge.
+
+        // FIXME:
+        // Coin chalenges β and γ for the permutation argument
+
+        // FIXME:
+        // Compute the accumulator for the permutation argument
```

Comment:
Adding also the absorption of the accumulator of the permutation argument + coining alpha.
@marcbeunardeau88 I know we discussed having only one global permutation argument, but it only works if we do have the same circuit over time, which might not be always the case.